html/themes/custom/common_design_subtheme/templates/user/form--user-form.html.twig

```diff
@@ -0,0 +1,50 @@
+{#
+/**
+ * @file
+ * Theme override for a 'form' element.
+ *
+ * Available variables
+ * - attributes: A list of HTML attributes for the wrapper element.
+ * - children: The child elements of the form.
+ *
+ * @see template_preprocess_form()
+ */
+#}
+{{ attach_library('common_design_subtheme/rw-form') }}
+
+<form{{ attributes }}>
+  <fieldset id="account">
+    <legend>{{ 'Account information'|t }}</legend>
+    {{ element.account.name }}
+    {{ element.account.current_pass }}
+    {{ element.account.mail }}
+    {{ element.account.pass }}
+    {{ element.account.reset_pass }}
+  </fieldset>
+
+  <fieldset id="details" class="details">
+    <legend>{{ 'Account status and note'|t }}</legend>
+    {{ element.account.status }}
+    {{ element.account.roles }}
+    {{ element.account.notes }}
+  </fieldset>
+
+  <fieldset id="actions">
+    <legend class="visually-hidden">{{ 'Form submission and other actions'|t }}</legend>
+    {{ element.actions }}
+  </fieldset>
+
+  {{
+    form|without([
```

Comment:
`form` is not defined. The user form is similar to the taxonomy one, so probably need the loop with the `skip_elements` rather than the `form|without`.

Reply:
@orakili 
When I add
```

  {%
    set skip_elements = [
    'name',
    'account.current_pass',
    'account.mail',
    'account.pass',
    'account.reset_pass',
    'account.status',
    'account.roles',
    'account.field_notes',
    'actions',
  ]
  %}

  {% for key, child in element if key|first != '#' %}
    {{ key not in skip_elements ? child }}
  {% endfor %}
```

The result is duplicate fields. I changed it to use `form` for this reason but I see it's not defined, as you said.
When I don't add anything below the fields I print directly, I get the intended result i.e. a single set of fields printing in the order as they are in template markup.